### PR TITLE
fix(): Change default endpoint to barracuda and update licenses

### PR DESCRIPTION
**Pull request by tse4a**
- set default endpoint to api.us.barracuda.com
- update licenses

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -115,11 +116,25 @@ func aliasNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedName {
...
+		authViper.Set(ckeyAuthEndpoint, DefaultEndpoint)
+		global.WriteFiles = true
+		if global.WriteFiles {
+			err := authViper.WriteConfig()
+			if err != nil {
+				fmt.Printf("Error writing auth file: %v\n", err)
+				return
+			}
+		}
+		fmt.Fprintf(os.Stderr, "WARNING: The endpoint in use is no longer valid and was replaced by the new default endpoint '%s'. If necessary, please update your endpoint with the 'cluster' command before logging in.\n", DefaultEndpoint)
```
Comment (oNaiPs) [inline cmd/root.go:135]:
```suggestion
		fmt.Fprintf(os.Stderr, "WARNING: The endpoint in use is deprecated and may stop working in the future. Please update your endpoint using 'access-cli cluster set us|eu' command before logging in.\n")
```
@tse4a i suggest we add to the warning the endpoint shortcuts to minimize mistakes while transitioning.

**Review by oNaiPs — DISMISSED** [review]

**Review by joaoestrela — DISMISSED** [review]

File: cmd/root.go (hunk 1 above)
Comment (tse4a) [reply cmd/root.go:135]: This warning is now obsolete. Updating the config automatically to point to respective new endpoint.

Comment (tse4a) [reply cmd/root.go]: @oNaiPs updated PR to automatically update existing endpoints.

**Review by oNaiPs — CHANGES_REQUESTED** [review]

**Review by oNaiPs — DISMISSED** [review]
i think you did a bad push since there's some code that should have been removed

File: cmd/root.go [hunk 2]
```diff
@@ -115,11 +116,32 @@ func aliasNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedName {
...
+		}
+		global.WriteFiles = true
+		if global.WriteFiles {
+			err := authViper.WriteConfig()
+			if err != nil {
+				fmt.Printf("Error writing auth file: %v\n", err)
+				return
+			}
+		}
+		fmt.Fprintf(os.Stderr, "INFO: The endpoint '%s' is deprecated and may stop working in the future. Your config has been automatically updated with '%s'. No further action is needed on your side.\n", endpoint, new_endpoint)
```
[the remark this answers is not in the document]
Comment (tse4a) [reply cmd/root.go:141]: removed

File: cmd/root.go [hunk 3]
```diff
@@ -115,11 +116,32 @@ func aliasNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedName {
...
+		new_endpoint := endpoint
+		if strings.Contains(endpoint, "enterprise.eu.fyde.com") {
+			new_endpoint = "api.eu.access.barracuda.com"
+			authViper.Set(ckeyAuthEndpoint, new_endpoint)
+		} else {
+			new_endpoint = DefaultEndpoint
+			authViper.Set(ckeyAuthEndpoint, new_endpoint)
+		}
+		global.WriteFiles = true
+		if global.WriteFiles {
```
[the remark this answers is not in the document]
Comment (tse4a) [reply cmd/root.go:134]: fixed

**Review by pablosole — DISMISSED** [review]

**Review by pablosole — APPROVED** [review]

**Pull request merged**